src/app/breadcrumbs/breadcrumbs-item/breadcrumbs-item.component.html

```diff
@@ -1,3 +1,4 @@
 <a class="breadcrumb-item-link" *ngIf="breadcrumb"
-	 href="{{breadcrumb.link}}"
+	 [routerLink]="breadcrumb.link"
```

Comment:
Breadcrumbs have to show an item name on an item page:
![image](https://user-images.githubusercontent.com/81073/71118036-59be6280-21e8-11ea-84d5-84b19b62c84b.png)